### PR TITLE
Improve-multiselect

### PR DESCRIPTION
**Pull request by stevecassidy**
Signed-off-by: Steve Cassidy <steve.cassidy@mq.edu.au>

# Improve Multiselect

## JIRA Ticket

See Miro!

## Description

Multiselect was a bit confusing, add checkboxes to the select to emphasise that more than one can be checked. 

## Proposed Changes

Updates MultiSelect to add checkboxes.
Updates Select to match the new MultiSelect implementation.

image

## How to Test

View the BSS survey, Hazards should show as the new multiselect with checkboxes.

## Checklist

- [x] I have confirmed all commits have been signed.
- [x] I have added JSDoc style comments to any new functions or classes.
- [x] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

File: app/src/gui/fields/select.tsx [hunk 1]
```diff
@@ -33,7 +41,44 @@ interface Props {
...
+export const Select = (props: Props & TextFieldProps) => {
+  const theme = useTheme();
+  const handleChange = (e: any) => {
+    props.form.setFieldValue(props.field.name, e.target.value, true);
+  };
+
+  return (
+    <FormControl sx={{m: 1, width: '100%'}}>
+      <InputLabel
+        id="multi-select-label"
```
Comment (PeterBaker0) [inline app/src/gui/fields/select.tsx:55]: did you want to change these IDs?

**Comment by PeterBaker0** [comment]
image

is this change in the colour of the helper text when the element is focused intentional?

**Comment by stevecassidy** [comment]
That didn't change in this PR. That's a RadioGroup field. No idea what's going on there though.

**Review by PeterBaker0 — CHANGES_REQUESTED** [review]
Looks good - wondering if we can wrap text for overflowing menu items e.g. 

image

Thanks

**Comment by PeterBaker0** [comment]
> That didn't change in this PR. That's a RadioGroup field. No idea what's going on there though.

ah - all good then I'll throw into a JIRA ticket (not miro!)

**Comment by stevecassidy** [comment]
image

**Review by PeterBaker0 — APPROVED** [review]
Lgtm - thanks for doing the word wrap

**Pull request merged**